e2etests/pkg/dump/k8s.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package dump
+
+import (
+	"log"
+	"strings"
+	"time"
+
+	frrk8sv1beta1 "github.com/metallb/frrk8s/api/v1beta1"
+	"github.com/openshift-kni/k8sreporter"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func NewK8sReporter(kubeconfig, namespace string) *k8sreporter.KubernetesReporter {
+	if ReportPath == "" {
+		log.Fatalf("ReportPath is not set")
+	}
+
+	// When using custom crds, we need to add them to the scheme
+	addToScheme := func(s *runtime.Scheme) error {
+		err := frrk8sv1beta1.AddToScheme(s)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	// The namespaces we want to dump resources for (including pods and pod logs)
+	dumpNamespace := func(ns string) bool {
+		switch {
+		case ns == namespace:
+			return true
+		case strings.HasPrefix(ns, "frrk8s"):
+			return true
+		}
+		return false
+	}
+
+	// The list of CRDs we want to dump
+	crds := []k8sreporter.CRData{
+		{Cr: &frrk8sv1beta1.FRRConfigurationList{}},
+	}
+
+	reporter, err := k8sreporter.New(kubeconfig, addToScheme, dumpNamespace, ReportPath, crds...)
+	if err != nil {
+		log.Fatalf("Failed to initialize the reporter %s", err)
+	}
+	return reporter
+}
+
+func K8sInfo(testName string, reporter *k8sreporter.KubernetesReporter) {
+	reporter.Dump(10*time.Minute, testName)
```

Comment:
can you please pass the 10 minutes as a const for clarity?
we named this as `LogsExtractDuration` on the cnf-tests.